src/Writers/HTMLWriter.jl

```diff
@@ -1748,7 +1748,43 @@ function mdconvert(d::Dict{MIME,Any}, parent; kwargs...)
     if haskey(d, MIME"text/html"())
         out = Documents.RawHTML(d[MIME"text/html"()])
     elseif haskey(d, MIME"image/svg+xml"())
-        out = Documents.RawHTML(d[MIME"image/svg+xml"()])
+        svg = d[MIME"image/svg+xml"()]
+
+        # The xmlns attribute has to be present for data:image/svg+xml
+        # to work (https://stackoverflow.com/questions/18467982)
+        svg_tag = match(r"<svg[^>]*>", svg).match
```

Comment:
You're right, I didn't think about this. The svg will be invalid then, but I've added an additional check to avoid a crash anyway. I'm now just returning the svg as is if it doesn't have a tag.